sdks/python/src/opik/evaluation/models/litellm_chat_model.py

```diff
@@ -54,11 +53,20 @@ def _check_must_support_arguments(self, args: Optional[List[str]]) -> None:
             if key not in self.supported_params:
                 raise ValueError(f"Unsupported parameter: '{key}'!")
 
-    def _check_params(self, params: Dict[str, Any]) -> None:
+    def _check_params(self, params: Dict[str, Any]) -> Dict[str, Any]:
+        valid_params = params
+
         for key in params:
-            if key not in self.supported_params:
+            if "response_format" not in self.supported_params:
+                LOGGER.debug(
+                    "This model does not support the response_format parameter and therefore the json schema will not be enforced."
+                )
+                valid_params.pop("response_format", None)
+            elif key not in self.supported_params:
                 raise ValueError(f"Unsupported parameter: '{key}'!")
```

Comment:
From now on we can rely on `_check_must_support_arguments` method, and this method will simply filter out arguments that are not supported. So, the error raising may be removed.
An example with `response_format` shows us that sometimes there might be arguments that are _nice to pass to the provider function, but not necessary_. 
FYI @japdubengsub